plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplit.java

```diff
@@ -46,6 +47,7 @@
     private final List<HostAddress> addresses;
     private final TupleDomain<DeltaLakeColumnHandle> statisticsPredicate;
     private final Map<String, Optional<String>> partitionKeys;
+    private final SplitWeight splitWeight;
```

Comment:
Nit: match the order from the constr, put this between addresses and statisticsPredicate